wiki/plugins/metadata/tables.py

```diff
@@ -23,7 +23,7 @@ class PTokenAnnotationTable(tables.Table):
     sentid = tables.Column(accessor='sentence', verbose_name='Sent ID')
     
     def render_exid(self, record):
-        return record.html()
+        return record.html
```

Comment:
Starting here, you switched to returning a function? Can you explain this line?

Reply:
In models.py, `CorpusSentence.html` has been made a property (via the `@cached_property` decorator).